File: sdk/swift/Sources/Microsandbox/Microsandbox.swift
/// A minimal SDK for the Microsandbox project.
public struct Microsandbox {
    /// Returns a greeting message for the given name.
    ///
    /// - Parameter name: The name to greet
    /// - Returns: A greeting message
    public static func greet(_ name: String) -> String {
        let message = "Hello, \(name)! Welcome to Microsandbox!"
        print(message)
        return message
    }
}

File: sdk/scala/src/main/scala/com/microsandbox/Microsandbox.scala
package com.microsandbox

/**
 * Microsandbox Scala SDK
 * A minimal SDK for the Microsandbox project.
 */
object Microsandbox {
  /**
   * Returns a greeting message for the given name.
   *
   * @param name The name to greet
   * @return A greeting message
   */
  def greet(name: String): String = {
    val message = s"Hello, $name! Welcome to Microsandbox!"
    println(message)
    message
  }
}
